components/LoadingPage/Loading.js

```diff
@@ -0,0 +1,86 @@
+import CPressed from "./Images/CPressed.svg";
+import HPressed from "./Images/HPressed.svg";
+import APressed from "./Images/APressed.svg";
+import PPressed from "./Images/PPressed.svg";
+import RedBirdPressed from "./Images/RedBirdPressed.svg";
+import CIdle from "./Images/CIdle.svg";
+import HIdle from "./Images/HIdle.svg";
+import AIdle from "./Images/AIdle.svg";
+import PIdle from "./Images/PIdle.svg";
+import RedBirdIdle from "./Images/RedBirdIdle.svg";
+
+import { useEffect, useState, useRef } from "react";
+
+export default function LoadingPage() {
+  const [time, setTime] = useState(0);
+  const [startTime, setStartTime] = useState(0);
+
+  const interval = useRef();
+
+  useEffect(() => {
+    setStartTime(Date.now());
+  }, []);
+
+  useEffect(() => {
+    interval.current = setInterval(() => {
+      setTime(() => Date.now() - startTime);
+    }, 1);
+  });
+
+  return (
+    <div
+      style={{
+        display: "flex",
+        justifyContent: "center",
+        height: "100vh",
+        width: "100vw",
+      }}
+    >
+      {time > 1000 && time < 1500 ? (
+        <CPressed
+          style={{ marginTop: "auto", marginBottom: "auto", width: "10%" }}
+        />
+      ) : (
+        <CIdle
+          style={{ marginTop: "auto", marginBottom: "auto", width: "10%" }}
+        />
+      )}
+      {time > 1600 && time < 2100? (
+        <HPressed
+          style={{ marginTop: "auto", marginBottom: "auto", width: "10%" }}
+        />
+      ) : (
+        <HIdle
+          style={{ marginTop: "auto", marginBottom: "auto", width: "10%" }}
+        />
+      )}
```

Comment:
I'm thinking that for these SVGs we should create a component called Key, with props such as letter (we can designate "M" to the McGill bird or just "RedBird") and position ("pressed" or "idle") and we can fetch the svg by just making a string literal of these props `${letter}${position}.svg`. Let me know your thoughts